src/Kris/LaravelFormBuilder/Fields/FormField.php

```diff
@@ -223,7 +227,7 @@ protected function prepareOptions(array $options = [])
             $this->addErrorClass();
         }
 
-        if ($this->getOption('required') === true) {
+        if ($this->getOption('required') === true || $rulesParser->containsRequired($rules)) {
```

Comment:
There is no need for special method on rules parser to check for that, you can only parse the rules, and do `in_array('required', $parsedRules)`.